apps/o2-blocks/src/github-issue-template/editor.js

```diff
@@ -0,0 +1,154 @@
+		<Path
+			fill="#fff"
+			d="M335 499c14 0 12 17 12 17H165s-2-17 12-17c13 0 16-6 16-12l-1-50c-71 16-86-28-86-28-12-30-28-37-28-37-24-16 1-16 1-16 26 2 40 26 40 26 22 39 59 28 74 22 2-17 9-28 16-35-57-6-116-28-116-126 0-28 10-51 26-69-3-6-11-32 3-67 0 0 21-7 70 26 42-12 86-12 128 0 49-33 70-26 70-26 14 35 6 61 3 67 16 18 26 41 26 69 0 98-60 120-117 126 10 8 18 24 18 48l-1 70c0 6 3 12 16 12z"
+		/>
+	</SVG>
+);
+
+const Shell = ( { as: El = 'div', className, title, subTitle, icon, body, ...elementProps } ) => {
+	return (
+		<El { ...elementProps } className={ classNames( 'wp-block-a8c-github-template', className ) }>
+			{ typeof icon === 'string' ? (
+				<div className="github-template__icon">{ icon }</div>
+			) : (
+				<Icon className="github-template__icon" icon={ icon || githubIcon } />
+			) }
+			<span className="github-template__title">{ title }</span>
+			<div className="github-template__sub-title">{ subTitle }</div>
+			{ body && <span className="github-template__body">{ body }</span> }
+		</El>
+	);
+};
+
+const Edit = ( {
+	onChangeTitle,
+	onChangeUserOrOrg,
+	onChangeRepoName,
+	onChangeBody,
+	title,
+	userOrOrg,
+	repo,
+	body,
+} ) => (
+	<Shell
+		className="is-edit"
+		title={ <TextControl placeholder="Issue Title" onChange={ onChangeTitle } value={ title } /> }
+		subTitle={
+			<>
+				<TextControl
+					placeholder="User or Organization"
+					onChange={ onChangeUserOrOrg }
+					value={ userOrOrg }
+				/>
+				&nbsp;/&nbsp;
+				<TextControl placeholder="Repository" onChange={ onChangeRepoName } value={ repo } />
+			</>
+		}
+		body={ <TextareaControl placeholder="Issue Body" onChange={ onChangeBody } value={ body } /> }
+	/>
+);
+
+const View = ( { title, userOrOrg, repo, ...rest } ) => (
+	<Shell
+		title={ `${ __( 'Create Issue:' ) } ${ title }` }
+		subTitle={ `${ userOrOrg }/${ repo }` }
+		{ ...rest }
+	/>
+);
+
+const Invalid = () => (
+	<Shell
+		className="is-warning"
+		title={ __( 'Please fill in the required fields' ) }
+		subTitle={ __(
+			'Title, Org, and Repository are required. Select this block to open the form and fill them.'
+		) }
+		icon="⚠"
+	/>
+);
+
+registerBlockType( 'a8c/github-issue-template', {
+	title: __( 'Github Issue Template', 'a8c' ),
+	icon: githubIcon,
+	category: 'layout',
+	attributes: {
+		userOrOrg: {
+			type: 'string',
+		},
+		repo: {
+			type: 'string',
+		},
+		title: {
+			type: 'string',
+		},
+		body: {
+			type: 'string',
+		},
+	},
+	edit: ( { setAttributes, attributes, attributes: { title, userOrOrg, repo }, isSelected } ) => {
+		const handlers = {
+			onChangeUserOrOrg( newUserOrOrg ) {
+				setAttributes( { userOrOrg: newUserOrOrg } );
+			},
+
+			onChangeRepoName( newRepo ) {
+				setAttributes( { repo: newRepo } );
+			},
+
+			onChangeTitle( newTitle ) {
+				setAttributes( { title: newTitle } );
+			},
+
+			onChangeBody( newBody ) {
+				setAttributes( { body: newBody } );
+			},
+		};
+
+		const isValid = title && userOrOrg && repo;
```

Comment:
I don't think title should be required. We can tweak things in another PR.

Reply:
I assumed that a title would be required otherwise if we only use a "Create Issue" fallback, the meaning would depend on the surrounding context (Create what issue?). But yep, we can change it afterward, I'm okay with that 👍🏻